### DISCUSSION TITLE
I am blocked from logining in

### DISCUSSION THREAD

**Discussion by davesalvador** [vercel/community#4085] [Account Issue] [answered]
### Question

This is my first time that I will use vercel and I am blocked I dont know why

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
